lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -225,9 +225,18 @@ SPIRVToLLVMDbgTran::transTypeArray(const SPIRVExtInst *DebugInst) {
     if (!getDbgInst<SPIRVDebug::DebugInfoNone>(Ops[I])) {
```

Comment:
It also would be nice to have an example of expected TypeArray representation in SPIR-V for each if block.

Reply:
Aren't they represented here? https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/test/DebugInfo/DebugInfoSubrange.ll

Reply:
Yes, thank you. But as I see DIExpression case is represented only for empty expression. It would also be nice to have non-empty expression case.